cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,17 @@
+import time
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("Waiting for db")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+            except OperationalError:
+                self.stdout.write("DB is not available...")
```

Comment:
There is a typo in the database readiness message. Consider changing 'DB is not available...' to 'Database is not available...' for clarity and consistency.